app/views/broadcast_announcements/_broadcast_announcement.html.erb

```diff
@@ -0,0 +1,14 @@
+<tr id="<%= dom_id broadcast_announcement %>">
+  <td><%= broadcast_announcement.message %></td>
+  <td><%= link_to  broadcast_announcement.link, broadcast_announcement.link %></td>
+  <td><%= User.find(broadcast_announcement.user_id).name %></td>
```

Comment:
You should be able to write `broadcast_announcement.user.name` here. Or if you want to be a bit more defensive, `broadcast_announcement.user&.name || "N/A"`.